addons/website_mail_channel/views/snippets.xml

```diff
@@ -1,52 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-
-<template id="remove_external_snippets" inherit_id="website.external_snippets">
-    <xpath expr="//t[@t-install='website_mail_channel']" position="replace"/>
-</template>
-
```

Comment:
Actually this file should have stayed to handle snippet removal and addition ;)

Reply:
Not sure I understand this one

Reply:
t-install and t-snippet should not have been moved in the dedicated snippet xml file but should have stayed here.

Reply:
Hm, thought u meant it was an error.
Alright, t-install and t-snippet should be inside `/views/snippets/snippets.xml`.
That file did not exists inside `website_mail_channel`, created it, should be good now